pyproject.toml

```diff
@@ -59,6 +59,7 @@ version_scheme = "post-release"
 
 [tool.mypy]
 plugins = ["pydantic.mypy"]
+#disallow_untyped_defs = true
```

Comment:
We need to uncomment this to actually test this in the CI.

Reply:
I know, but I have not completed all type annotations.